tests/remote/test_remote_decode.py

```diff
@@ -176,18 +177,6 @@ def test_output_type_pt_partial_postprocess_return_type_pt(self):
             f"{output_slice}",
         )
 
-    def test_do_scaling_deprecation(self):
```

Comment:
Wan VAE doesn't have a `scaling_factor` that we could pass, so this deprecation/removing `do_scaling` doesn't work, we will keep it.